### DISCUSSION TITLE
Accidentally removed pacman from my MSYS2 while installing git for windows

### DISCUSSION THREAD

**Discussion by pjujeczko** [git-for-windows/git#3135] [Q&A]
I was carelessly doing steps from https://github.com/git-for-windows/git/wiki/Install-inside-MSYS2-proper. I didn't run the first bunch of commands (points 1-5) in MSYS shell, instead I used the MinGW64 one. I've ended up without pacman in my MSYS2. How do I restore it? I couldn't find anything relevant in the issues.

### Setup
I use Win10, git shows version 2.31.0, i.e. the one that comes from Msys2. Before the installation everything in my MSYS2 was up to date.

**Comment by dscho** [comment]
As per the first paragraph in the linked wiki page, I'll summon @elieux:

> If you need help with the installation steps, you can contact @Elieux on the git-for-windows/git Gitter room or on the #msys2 IRC channel. Please don't file issues with Git for Windows installed this way until you verify them in an official Git for Windows distribution.
